phoenicis-configuration/src/main/java/org/phoenicis/configuration/localisation/PropertiesResourceBundle.java

```diff
@@ -0,0 +1,24 @@
+package org.phoenicis.configuration.localisation;
+
+import java.util.Enumeration;
+import java.util.Properties;
+import java.util.ResourceBundle;
+
+public class PropertiesResourceBundle extends ResourceBundle {
```

Comment:
This could be quite useful, when we want to provide translations inside the repositories right?